tests/unit/test_unit_speech_commands.py

```diff
@@ -86,6 +86,10 @@ def test_pytorch_audio_dataset_with_perturbation(self):
         perturbations = [
             perturb.WhiteNoisePerturbation(min_level=-90, max_level=-46),
             perturb.ShiftPerturbation(min_shift_ms=-5.0, max_shift_ms=5.0),
+            perturb.TimeStretchPerturbation(min_speed_rate=0.9, max_speed_rate=1.1),
+            perturb.SpeedPerturbation(
+                sr=self.featurizer_config['sample_rate'], min_speed_rate=0.9, max_speed_rate=1.1
+            ),
```

Comment:
Can you test each perturbation separately and add asserts that check they do what they are supposed to?

Reply:
Hmm for shift I can probably do a strider check. For white noise, do I just check the two items are not exactly the same Element wise ?

Reply:
I am not too sure. Maybe do element-wise perturbed - unperturbed. And then fit a guassian on it? And see if the mean and std is what you expect?

Reply:
Added individual checks for each of the augmentations.